service/contact_service.go

```diff
@@ -13,6 +13,23 @@ func SaveContact(ctx context.Context, cont dto.Contact) error {
 	return dal.SaveContact(ctx, contact)
 }
 
+func GetContact(ctx context.Context) ([]byte, error) {
```

Comment:
Don't return byte array here. 
Please convert entity (which returns from dal) to DTO and return object of DTO.